package.json

```diff
@@ -222,6 +222,7 @@
     ]
   },
   "resolutions": {
+    "@microsoft/api-extractor/typescript": "4.1.5",
```

Comment:
I reported this some time ago https://github.com/microsoft/rushstack/issues/2565

wondering who can we reach out to to get proper fix ? 
@ecraig12345 @JustSlone